### ISSUE TITLE
Unable to adjust sliders with a screenreader.

### ISSUE THREAD

**Issue by blindndangerous** [teticio/Deej-AI#76]
Hello, just found this project and set it up.  However, when accessing the webpage to start, the sliders for Keep On and Drunk are not accessible to those who use a screenreader.  I use nvda as my screenreader, and the only things that I can see are several clickable objects and the numbers for the start and end of the sliders, but no way to adjust them.  When using object navigation, I see the NVDA says Diagram where I think the sliders should be.

**Comment by teticio** [comment]
Ah I am sorry about that. It is probably a limitation of the Dash widgets and the project is frozen to an old version... I don't plan to update it as it is just a demo. But maybe you can use my https://deej-ai.online project, although it is slightly different.
